docs/pages/guides/using-sentry.mdx

```diff
@@ -193,6 +193,27 @@ export default Sentry.wrap(App);
 
 </Tabs>
 
+> **info** Additionally, if your app is using EAS Update, we recommend configuring your Sentry release with the
+> following format to see related Sentry errors in Expo dashboard with the Sentry integration enabled:
```

Comment:
```suggestion
Additionally, if your app is using EAS Update, we recommend configuring your Sentry release with the following format to **see related Sentry errors in the Expo dashboard with the Sentry integration enabled**:
```

Let's avoid adding a callout here since the code block below relates to it, and having this info in the callout makes the code block appear below, somewhat isolated (and not a good pattern IMO).

Also, let's make the last part of the sentence bold to ensure that the reader pays attention.